test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -97,14 +119,14 @@ TEST_F(XdsOverrideHostTest, SwapChildPolicy) {
             absl::OkStatus());
   for (absl::string_view address : kAddresses) {
     auto subchannel = FindSubchannel({address});
-    ASSERT_TRUE(subchannel);
+    ASSERT_NE(subchannel, nullptr);
     ASSERT_TRUE(subchannel->ConnectionRequested());
     subchannel->SetConnectivityState(GRPC_CHANNEL_CONNECTING);
     subchannel->SetConnectivityState(GRPC_CHANNEL_READY);
     EXPECT_NE(ExpectState(GRPC_CHANNEL_READY), nullptr);
```

Comment:
Please check the behavior of the picker instead of just asserting that it's non-null.